mkdocs.yml

```diff
@@ -88,7 +89,10 @@ plugins:
         python:
           paths: ["."]
           options:
+            heading_level: 1
             show_root_heading: true
+            show_root_members_full_path: true
+            show_source: false
```

Comment:
Look below for:

```
plugins:
  - "search"
  - "mkdocs-version-annotations"
```

and change to

https://github.com/nautobot/nautobot/blob/ba2b3028566b1995f4ca868404f8e6aec116d458/mkdocs.yml#L82-L84

Reply:
I'm really not following what you mean on this.

Reply:
@itdependsnetworks can you expand. The linked info doesn't seem to match what the reference code is.